definitions/npm/@testing-library/react_v8.x.x/flow_v0.67.1-v0.103.x/react_v8.x.x.js

```diff
@@ -1,4 +1,16 @@
 declare module '@testing-library/react' {
+  // This type comes from
+  // https://github.com/facebook/flow/blob/v0.103.0/lib/react-dom.js#L64
+  declare type ReactDOMTestUtilsThenable = {
+    then(resolve: () => mixed, reject?: () => mixed): mixed,
+    ...
```

Comment:
Inexact objects syntax was added later than `0.67.1`, just remove it, it fine to only have it in `0.104.x-`.

Reply:
Thanks for pointing that out! Pushed a new commit that removes the inexact object syntax.